### ISSUE TITLE
Links only work when ending in a slash

### ISSUE THREAD

**Issue by JanoschDeurer** [orzih/mkdocs-with-pdf#21]
If I add a markdown Link to a chapter like this
```
[My Chapter](../../my/chapter)
```
it is not resolved in the pdf and I get an error on build time that there is no anchor, however the following works:
```
[My Chapter](../../my/chapter/)
```
Both are resolved correctly in the web version.

**Comment by orzih** [comment]
Thanks, I fixed it in v0.7.4.

**Comment by JanoschDeurer** [comment]
Perfect :) Thanks again :)
